foodsaving/stores/models.py

```diff
@@ -45,7 +57,7 @@ class PickupDateSeries(BaseModel):
         related_name='series',
         on_delete=models.CASCADE
     )
-    max_collectors = models.PositiveIntegerField(blank=True, null=True)
+    max_collectors = models.Posit iveIntegerField(blank=True, null=True)
```

Comment:
it's already fixed but I created a new problem so I didn't want to upload the changes...